docs/roadmap.md

```diff
@@ -19,12 +19,33 @@ The following roadmap gives an overview of what we are currently working on and
 
 *Status*: tech spec out
 
-#### Integrate with data quality system
+#### Integrate with Data Quality system
 *What*: Integrate with different data quality systems to provide quality score.
 
 *Status*: planning
 
 ## Mid Term - Our Next steps
+#### Improve search ranking
+*What*: Update search ranking to be informed by "badges" that may exist on data sets e.g. deprecated, etc.
+
+*Status*: planning
+
+#### Notifications when a table evolves
+*What*: Notify users in Amundsen (akin to Facebook notifications or similar) when a table evolves. Owners of data and consumers of data will likely need to be notified of different things.
+
+*Status*: planning has not started 
+
+#### Commonly joined tables / browsing the data model
+*What*: As a data user, I would like to see commonly joined tables and how to join them.
+One option would be to show commonly joined tables and showing example join queries. Another option would be to provide a navigational experience for data model, showing foreign keys and which tables they come from.
+
+*Status*: planning has not started 
+
+#### Curated navigation experience
+*What*: Currently Amundsen's experience is very focussed on search. However, especially for new users, an experience where they are able to navigate through the data hierarchy is very important. This item proposes to revamp the navigational experience in Amundsen (currently, barebones - based on tags) to do justice to the user need to browse through data sets when they don't know what to even search for.
+
+*Status*: planning
```

Comment:
cc @danwom

Reply:
Yup, this is exactly the item @danwom has been thinking about, I just wanted an open source roadmap placeholder for it.